JavaScript/.eslintrc.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+    env: {
+        browser: true,
+    },
+    extends: [
+        'airbnb',
+        'plugin:jsdoc/recommended',
+    ],
+    rules: {
+        'max-len': ['error', {
+            'code': 120,
+            'comments': 120,
+            'tabWidth': 4,
+            'ignoreUrls': false,
+            'ignoreTrailingComments': false,
+            'ignoreComments': false
+        }],
+        indent: ['error', 4, {
+            SwitchCase: 1,
+        }],
+        'jsdoc/require-file-overview': 1,
```

Comment:
not sure if this is needed here
as if this is useful very rarely, because in ts we have one class - one file and so the description of the file = description of the class
i.e. it could be useful when you describe some utils, a file with common logic or index with imports for example

Reply:
ok, let's remove it from the default config